### DISCUSSION TITLE
mesh plot phoebe

### DISCUSSION THREAD

**Discussion by kittipongwangnok** [phoebe-project/phoebe2#988] [General]
How can we plot the model configuration at phase = 0.0, 0.25, 0.5, 0.75, and 1.0 in PHOEBE? Can we produce the plot at any phase? Moreover, I am just curious, what do the u, and w in the plot represent?
![Screenshot 2567-11-26 at 3 27 22 PM](https://github.com/user-attachments/assets/5abaa4dc-2952-4413-8f60-7a9359f15ceb)

**Comment by kecnry** [comment]
So that we can improve the documentation - which tutorials/example scripts did you use and how could we improve them to include the information you are looking for?

**Reply by kittipongwangnok** [reply]
I would like to plot the orbital configuration of the binary model that is related to the phase of the light curve. Right now, I only can plot model that I have shown before.

**Reply by kecnry** [reply]
Right, I'm trying to understand how we can improve the documentation so users can find the information like this easier.  Are there any specific ones you've looked at to get this far but don't point to the right information?

**Reply by kittipongwangnok** [reply]
No, that is all for me right now. The nice example provided me with a lot of information. Thank you very much for providing these materials.

**Reply by kecnry** [reply]
https://phoebe-project.org/docs/latest/tutorials/meshes and https://phoebe-project.org/docs/latest/tutorials/MESH should cover everything that you're looking for - if you want phases instead of times, just use `compute_phases` when adding the dataset (see https://phoebe-project.org/docs/latest/tutorials/compute_times_phases for more details).  If anything isn't clear in these tutorials or they were hard to find as the right place to look, please do let us know so that we can make improvements to them.

**Comment by kittipongwangnok** [comment]
Thank you very much. Right now, I can produce the plot at any orbital phase. 
![Screenshot 2567-11-27 at 2 01 37 PM](https://github.com/user-attachments/assets/e3ed3117-defa-422f-955e-567c7d2ecac4)
